### PR TITLE
Build manifest for org.gnome.gedit

### PR DESCRIPTION
**Pull request by ltilve**

### CODE REVIEW

File: org.gnome.gedit.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    "sha256": "442f88fecd7f9e00a6a9d0af2d762744dd3c119217ff11d6739500f75b0f10be"
+                }
+            ]
+        },
+        {
+            "name": "gedit",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gedit/3.22/gedit-3.22.0.tar.xz",
```
Comment (alexlarsson) [inline org.gnome.gedit.json]: There seems to be a 3.22.1 release out now.

File: org.gnome.gedit.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                }
+            ]
+        },
+        {
+            "name": "gtksourceview",
+            "config-opts": ["--disable-Werror" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.1.tar.xz",
```
Comment (alexlarsson) [inline org.gnome.gedit.json]: 3.24.3 is out now

**Comment by alexlarsson** [comment]
Building here:
https://flathub.org/builds/#/builders/1/builds/138

**Comment by cosimoc** [comment]
This built fine -- closing this PR.

**Pull request closed without merging**